.github/workflows/emberjs-addons-ci.yml

```diff
@@ -0,0 +1,108 @@
+# GitHub Actions documentation:
+# https://docs.github.com/en/actions
+
+name: CI
+
+on:
+  workflow_call:
+    inputs:
+
+      nodejs_version:
+        description: Version of Node.js to use.
+        default: '12'
+        required: false
+        type: string
+
+      # Currently, Arrays cannot be passed as inputs so the trick is to pass an Array as a string
+      # and to decode the string with `fromJSON()` to rebuild the real Array.
+      # https://colinsalmcorner.com/musings-on-reusable-workflows/#array-hack
+      ember_try_scenarios:
+        description: Custom ember-try scenarios to run.
+        default: "[
+          'ember-lts-3.20',
+          'ember-lts-3.24',
+          'ember-release',
+          'ember-classic',
+          'ember-default-with-jquery',
+          'embroider-safe',
+          'embroider-optimized',
+        ]"
+        required: false
+        type: string
+
+jobs:
+  lint_tests:
+    name: Lint and tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          cache: yarn
```

Comment:
We should be possible to override this option.
Probabbly use npm by default and let the integration specify "yarn"

Reply:
We can indeed, do we want it in this first version? As all our repo use `yarn`.

Reply:
right , I'm fine with it tbh :) at least add it in inputs with yarn by default ? just to be future proof